chapter2/tests/t19/.gitignore

```diff
@@ -0,0 +1,19 @@
+__pycache__
+.git-crypt/
+.idea/
+.ssh/
+.terraform/
+.vscode
+*.bak
+*.pyc
+*.tfstate
+*.tfstate.backup
+*.zip
+.git_rev
+*.pem
+*.pub
+*.log
+assets/ssl
+builds/
+packer_cache
+git-crypt-key
```

Comment:
Also, remove files from the other task. Always make sure you don't have redundant files in your PR.

Reply:
> Also, remove files from the other task. Always make sure you don't have redundant files in your PR.

@roman-mightydevops 
Done